src/dos/cdrom.h

```diff
@@ -117,7 +121,8 @@ class CDROM_Interface_SDL : public CDROM_Interface
 	virtual bool	PlayAudioSector		(unsigned long start,unsigned long len);
```

Comment:
clean up the formatting for this class now that we've touched it

Reply:
I think the long term plan is to remove this class, as it's problematic for SDL2, so I wouldn't bother ;)

Reply:
Good stuff!